repos/system_upgrade/el7toel8/actors/redhatsignedrpmcheck/libraries/library.py

```diff
@@ -30,6 +31,8 @@ def generate_report(packages):
         remediation=remediation,
         severity='high',
     )
+    if is_verbose():
+        api.show_message(summary)
```

Comment:
what about using log.info here, since it would do the same but also give us info about where was this message produced
other option would be using some prefix or something to make this more visible to user, since show_message doesn't have any

Reply:
I tried some experiments with log and adding prefix.

But I think the default behaviour is good enough look:

```
2019-04-08 17:20:23.795 INFO     PID: 20311 leapp.workflow.Checks: Executing actor red_hat_signed_rpm_check
The following packages have not been signed by Red Hat and may be removed in the upgrade process:
- leapp
- leapp-deps
- leapp-repository
- leapp-repository-deps
- leapp-repository-sos-plugin
- python2-leapp
- snactor
```

Is your concern about which Actor you printed on the output?